src/Sentry/Internal/Hub.cs

```diff
@@ -757,6 +757,25 @@ public SentryId CaptureCheckIn(
         return SentryId.Empty;
     }
 
+    internal bool CaptureAttachment(SentryId eventId, SentryAttachment attachment)
```

Comment:
// Internal because e e give preference to attacking files directly to the event, using the scope (global or local to the capture function call).  ur we could consider making this public if more use cases come up.

Reply:
follow-up question:
Due to `<InternalsVisibleTo Include="Sentry.Unity" />` we may change something in `sentry-dotnet` that will break `sentry-unity`.

I'm wondering how careful we should be in general about changing `internal` types and members,
since we may notice the break of `sentry-unity` only when we update to the latest `sentry-dotnet`.

🤔